packages/evm/src/precompiles/bls12_381/mcl.ts

```diff
@@ -7,7 +7,7 @@ import {
   unprefixedHexToBytes,
 } from '@ethereumjs/util'
 
-import { ERROR, EvmError } from '../../exceptions.ts'
+import { EVMError, EvmError } from '../../errors.ts'
```

Comment:
Again curious about that dual naming. IMO these can't coexist, our users will inevitably get confused

Reply:
Yes I agree. So the first is a dict of the error strings, and the second is the actual constructor. Not sure how to rename this. Can we add this as some kind of static object to `EvmError`?

Reply:
I've resolved this for now by renaming `EVMError` to `EVMErrorMessages`, can look into adding the messages as a static field in a followup to avoid adding to many changes in this PR.